pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/ThreadRuntime.java

```diff
@@ -114,6 +114,18 @@ public void stop() {
         return CompletableFuture.completedFuture(javaInstanceRunnable.getAndResetMetrics());
     }
 
+    @Override
```

Comment:
We need to add the same logic for python as well

Reply:
can we do this for java first and add to python in subsequent PR?